front/app/modules/commercial/native_surveys/components/NativeSurveyForm/NativeSurveyForm.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { JsonForms } from '@jsonforms/react';
+
+// components
+import { Box, Title } from '@citizenlab/cl2-component-library';
+
+// renderers
+import CLCategoryLayout, {
+  clCategoryTester,
```

Comment:
Is there a nicer way to do imports like this (when there are so many)?

Reply:
We could create an index file within the controls folder that imports these components (Controls) and exports them. I have found that to work well in the past.